src/chocolatey/infrastructure.app/domain/PackageOrder.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright © 2017 - 2018 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Linq;
+
+namespace chocolatey.infrastructure.app.domain
+{
+    public enum PackageOrder
+    {
+        /// <summary>
+        /// Sort by package ID (also commonly referred to as name), in ascending order.
+        /// </summary>
+        Id,
+
+        /// <summary>
+        /// Sort by package title in ascending order.
+        /// </summary>
+        Title,
+
+        /// <summary>
+        /// Sort by popularity, with the most popular packages first.
+        /// </summary>
+        Popularity,
+
+        /// <summary>
+        /// Sort by last published date, from newest to oldest.
+        /// </summary>
+        LastPublished,
+
+        /// <summary>
+        /// Do not sort; return packages in the order received from the source.
+        /// </summary>
+        Unsorted,
+
+        /// <summary>
+        /// Alias for <see cref="Popularity"/>. Sort by download count.
+        /// </summary>
+        DownloadCount = Popularity,
+
+        /// <summary>
+        /// Alias for <see cref="Unsorted"/>. Sorting is determined by the server.
+        /// </summary>
+        Server = Unsorted

```

Comment:
Why the need for the alias?  Doesn't that complicate things unnecessarily?

Reply:
The aliases has been removed.